script.cinder/addon.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.cinder" name="Cinder" version="1.0.0" provider-name="Justin Rush">
+	<requires>
+		<import addon="xbmc.python" version="2.20.0"/>
```

Comment:
You've asked this to be pulled into the Jarvis repo. The correct version for xbmc.pyhton is 2.24.0, if that is the case.